tests/models/test_decoders.py

```diff
@@ -431,6 +447,7 @@ def test_common_shapes(
     # we don't currently support inferring gptq from get_model, so we must use an adapter with hf_configured
     gptq_kwargs_aiu, gptq_kwargs_cpu = __maybe_get_gptq_kwargs(model_path)
     is_gptq = len(gptq_kwargs_aiu) != 0
+    is_fp8 = "fp8" in ATTN_NAME
```

Comment:
this should be made a little more flexible, we might be asking for a FP8 model without fp8 attention, but it's good for a first version, as for today we are testing with FP8 attention